addons/vehicles/stringtable.xml

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
BOM Change? Didn't we already fix the BOM's of all stringtables?

Reply:
I save stringtables with tabler without checking later for BOM changes.